FileMetadataReading.py

```diff
@@ -0,0 +1,31 @@
+from datetime import datetime
+import os
+from FileDiscovery import FileDiscovery
+
+
+class FileMetadateRading:
+
+    def __init__(self, FiltratedListOfPhotos):
+        self.list_of_photos = FiltratedListOfPhotos
+
+    def read_modification_date_time(self):
+        time_md = [datetime.fromtimestamp(os.path.getmtime(i))
+                   for i in self.list_of_photos]
+        self.time_md_date_and_time = [
+            str(i.date()) + '-' + str(i.time()) for i in time_md]
+        return self.time_md_date_and_time
```

Comment:
return type of this method is concatenated string from date and time - unless you want to present this information to user it would be better to pass it in Data Transfer Object(DTO) - this way when you will need to use part of this result you  will be able to simply load it from class field instead of splitting string